src/data/logos.ts

```diff
@@ -42,7 +42,7 @@ export const logos = LogoCheck({
 	contentful: { file: 'contentful.svg', padding: '.05em' },
 	directus: { file: 'directus.svg', padding: '0 .1em' },
 	ghost: { file: 'ghost.png', padding: '.125em' },
-	'netlify-cms': { file: 'netlify-cms.svg', padding: '.1625em' },
+	'decap-cms': { file: 'decap-cms.svg', padding: '0 .1625em' },
```

Comment:
Small nit on the padding value here. This is a tricky logo to get to feel centred because of the off-centre top of the D, but this feels better to me:
```suggestion
	'decap-cms': { file: 'decap-cms.svg', padding: '0 .225em 0 .26em' },
```